src/main/java/address/controller/RootLayoutController.java

```diff
@@ -56,13 +75,50 @@ public void setShortcuts(){
         menuFileSaveAs.setAccelerator(ShortcutsManager.SHORTCUT_FILE_SAVE_AS);
     }
 
+    @Subscribe
+    private void handleSaveLocationChangedEvent(SaveLocationChangedEvent e) {
+        updateSaveLocDisplay();
+    }
+
+    @Subscribe
+    private void handleMirrorLocationChangedEvent(MirrorLocationChangedEvent e) {
+        updateMirrorLocDisplay();
+    }
+
+    private void updateSaveLocDisplay() {
```

Comment:
`Loc` and `Location` both used in method names. Standardize on one or the other? The longer one is preferred.